### DISCUSSION TITLE
MTS Features with Meros LAN

### DISCUSSION THREAD

**Discussion by 3lbart0** [krahabb/meross_lan#320] [Q&A]
Hello everyone,

I am interested in remplacing my old thermostat by Meros MTS200.

I did not find the list of actions possible by Meross LAN on such device.

My priority is to be able from HA, by keeping my devices paired with the offical Meross App :

- either to switch on/off the thermostat
- or enable/disable the "economy mode" 

Is that possible ?
Thank you for you support

**Comment by krahabb** [comment]
Hello @3lbart0 ,
meross_lan actually supports almost all features of mts200 (except managing the schedule)

**Comment by BlackDragon7402** [comment]
Hi @3lbart0,
maybe this helps you out (Sorry some parts of it are in German):
![grafik](https://github.com/krahabb/meross_lan/assets/119086185/8c670504-10be-4bd6-b666-9d30696701da)

I would describe it like that:
You can basically control anything more than HomeKit or even the Merros App ever will but you are missing 2 features:
- Managing of schedules like already mentioned (avaible in Meross App)
- Child Lock Functionality (avaible in Meross App)

**Reply by Clood82** [reply]
As i can see the summer mode is missing aswell. You can change it in the app, but not in HA. It would be awesome because I like to cool with it during summer.

**Reply by 3lbart0** [reply]
hi @BlackDragon7402 
many thanks for the reply, appreciated.
I have now 4 MTS 200 and this is working very well.

**Reply by BlackDragon7402** [reply]
@Clood82 Only able to cool with mine, therefore totally forgot about this feature.

@3lbart0 Great to here. Started to love the devices a lot because of there simplicity but still feature richness.
And with one of the last firmware upgrade the disconnect bug has been fixed and even "upgraded" the functionality by showing the temperature in 0.1 degree steps
